integration_tests/src/main/python/date_time_test.py

```diff
@@ -214,3 +214,36 @@ def test_string_unix_timestamp(data_gen, date_form):
 def test_date_format(data_gen, date_format):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark : unary_op_df(spark, data_gen).selectExpr("date_format(a, '{}')".format(date_format)))
+
+unsupported_date_formats = ['F']
+@pytest.mark.parametrize('date_format', unsupported_date_formats, ids=idfn)
+@pytest.mark.parametrize('data_gen', date_n_time_gens, ids=idfn)
+@pytest.mark.xfail(reason='cuDF does not support these formats')
```

Comment:
Why xfail and not test that we fall back to the CPU?

Reply:
Thanks, this is fixed now.